_posts/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,382 @@
+* [The Next Generation of RSEs](https://research-it.manchester.ac.uk/news/2023/10/24/the-next-generation-of-rses/) by Emma Simpson
+* [Diversity and inclusion in research software: Insights from an international workshop](https://urssi.us/blog/2023/10/26/diversity-and-inclusion-in-research-software-insights-from-an-international-workshop/) by Eric A. Jensen and Daniel S. Katz
+* [What we learned about research software careers from a recent international workshop](https://urssi.us/blog/2023/10/24/what-we-learned-about-research-software-careers-from-a-recent-international-workshop/) by Eric A. Jensen and Daniel S. Katz
+* [Code for Thought podcast: Being a Research Software Engineer in France - Ghislain Vaillant](https://codeforthought.buzzsprout.com/1326658/14124873-en-being-a-research-software-engineer-in-france-ghislain-vaillant) by Peter Schmidt
+* [Code for Thought podcast: Culture of Women in Tech (Diversity in Tech, Part 2) - Mariann Hardey](https://codeforthought.buzzsprout.com/1326658/14083061-en-culture-of-women-in-tech-diversity-in-tech-part-2-mariann-hardey) by Peter Schmidt
+* [Code for Thought podcast: ByteSized RSE: Web Development with Django](https://codeforthought.buzzsprout.com/1326658/14092409-en-bytesized-rse-web-development-with-django) by Peter Schmidt
+
+
+<a name="outreach"></a>
+# **US-RSE Outreach**
+
+The Outreach Working Group is actively recruiting new co-chairs to take over its leadership! Volunteers are also needed to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles span from organizer to reviewer to speaker and more. Here are some key projects that recent co-chairs, Miranda Mundt and Josh Teves, have led to benefit the overall US-RSE community:
+* Strategically planning and delivering the first-ever US-RSE conference in 2023
+* Organizing the submission of collaborative proposals to relevant conferences
+* Providing awesome SWAG to members
+* Assisting the overall US-RSE community with delineating between membership, communications, and outreach efforts
+
+
+If you are interested in contributing to a US-RSE hosted event, please reach out to the working group via the [#wg-outreach](https://app.slack.com/client/T8ZT4PJSW/CNR6E0N75) slack channel or [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org).
+
+The Funders Talk Series will host [Ashley Sands from the Institute of Museum and Library Services](https://us-rse.org/events/2024/2024-01-funder-talk-series/) on January 29th, 2024. Slides and a video of the talk will be posted afterwards. If you have suggestions for future talks, please contact [Dan Katz](mailto:d.katz@ieee.org).
```

Comment:
since this was already mentioned earlier in the newsletter, maybe point to that mention?

Reply:
I will most likely remove the Funders Talk blurb from here. However, I can add the last line with your contact information to the Funders Talk section of the newsletter.

Reply:
ok, thanks

Reply:
Thanks! We fixed/changed in latest commit!